azure-automation-plus-dsc-lab/nested/14.17.00.createDevServer.json

```diff
@@ -0,0 +1,139 @@
+﻿{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "devPrefix": {
+      "type": "string"
+    },
+    "azureRegion": {
+      "type": "string"
+    },
+    "adminUserName": {
+      "type": "string"
+    },
+    "adminPassword": {
+      "type": "securestring"
+    },
+    "devAvSetId": {
+      "type": "string"
+    },
+    "dev01nicId": {
+      "type": "string"
+    },
+    "saName": {
+      "type": "string"
+    },
+    "domainName": {
+      "type": "string"
+    },
+    "domainToJoin": {
+      "type": "string"
+    },
+    "domainJoinOptions": {
+      "type": "string"
+    }
+  },
+  "variables": {
+    "dev01name": "[concat(parameters('devPrefix'),1)]",
+    "devVmSize": "Standard_A1_v2",
+    "imagePublisher": "MicrosoftVisualStudio",
+    "imageOffer": "VisualStudio",
+    "imageSku": "VS-2017-Comm-Latest-WS2016",
+    "diskNameOs": "[toLower(concat(variables('dev01name'),'-syst'))]",
+    "saType": "Standard_LRS",
+    "diskNameData": "[toLower(concat(variables('dev01name'),'-data'))]",
+    "saUrlSuffix": ".blob.core.windows.net",
```

Comment:
See [BP guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#referencing-resource-properties) for how to handle storage account uris